config/samples/oran.openshift.io_v1alpha1_clustertemplate.yaml

```diff
@@ -236,24 +122,17 @@ spec:
                 description: Additional node-level annotations to be applied
                   to the rendered templates
                 type: object
+              extraLabels:
+                additionalProperties:
+                  additionalProperties:
+                    type: string
+                  type: object
+                description: Additional node-level labels to be applied to the
+                  rendered templates
+                type: object
               hostName:
                 description: Hostname is the desired hostname for the host
                 type: string
-              ignitionConfigOverride:
-                description: Json formatted string containing the user overrides
-                  for the host's ignition config IgnitionConfigOverride enables
-                  the assignment of partitions for persistent storage. Adjust
-                  disk ID and size to the specific hardware.
-                type: string
-              installerArgs:
-                description: Json formatted string containing the user overrides
-                  for the host's coreos installer args
-                type: string
-              ironicInspect:
-                default: ""
-                description: IronicInspect is used to specify if automatic introspection
-                  carried out during registration of BMH is enabled or disabled
-                type: string
               nodeLabels:
```

Comment:
nodeLabels is used for node labeling through BMH `annotation` - `bmac.agent-install.openshift.io.node-label.<node-label>: ""`. In theory, this can be achieved with `extraAnnotations`, but I think we can keep this dedicated field for specific node labeling purpose for less confusion.

Reply:
OK - sounds good